src/components/results/sensitivityAnalysis/sensitivity-analysis-result.js

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import PropTypes from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+import React, { useCallback, useEffect, useMemo, useRef } from 'react';
+import { CustomAGGrid } from '../../dialogs/custom-aggrid';
+import { useTheme } from '@mui/styles';
+
+function makeColumn(isNum, labelId, key, intl) {
+    return {
+        headerName: intl.formatMessage({ id: labelId }),
+        field: key,
+        numeric: isNum,
+        fractionDigits: isNum ? 4 : undefined,
+        comparator: () => 0,
+    };
+}
+
+function makeColumns(nOrNkIndex, functionTypeIndex, intl) {
+    const ret = [];
```

Comment:
rename to table or returnedTable be better ?

Reply:
Done, thank you